### DISCUSSION TITLE
How to do more than 18 keys - follow up

### DISCUSSION THREAD

**Discussion by sonicfan888** [Leather128/LeathersFunkinEngine#266] [Show and tell]
this is an follow up to #177 
5) Edit the mania txt files in the ui skin/default and pixel. (Removed hurt and warning cuz look at leather's comment)
Have fun!!!!
this is also an fix to the invisible key bug when adding more keys

**Comment by Leather128** [comment]
you don't need to edit the arrow type files though? wtf u talking about it's just ui skins

**Reply by sonicfan888** [reply]
so you don't need to edit the hurt and warning.
also i mean editing the binds, animation directions, mania gaps, mania note size, all that stuff
